hack/validate-blocked-edges.py

```diff
@@ -3,6 +3,7 @@
 import os
 import re
 import util
+from deepdiff import DeepDiff
```

Comment:
Do we need this?  It seems like we could construct the diff logic we need without having to move beyond the stdlib, and the closer we can stay to the stdlib, the easier maintence is.

Reply:
Done.